### PR TITLE
fix(examples) updated broken example link

### PR DESCRIPTION
**Pull request by eddiejaoude**

### CODE REVIEW

File: packages/conventional-changelog-cli/README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ To fully customize the tool, please checkout [conventional-changelog](https://gi
 
 ## Example output
 
-- https://github.com/ajoslin/conventional-changelog/blob/master/CHANGELOG.md
+- https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog/CHANGELOG.md
```
Comment (hbetts) [inline packages/conventional-changelog-cli/README.md]: @eddiejaoude it looks like you originally were pointing to the `cli` package, but switched to the normal `conventional-changelog` package. Any particular reason? (I would assume the `cli` package would point to its own `CHANGELOG.md` file as an example)

Comment (eddiejaoude) [reply packages/conventional-changelog-cli/README.md]: @hbetts the original repo link `ajoslin/conventional-changelog` redirects to `conventional-changelog/conventional-changelog`, so I used the changelog from `conventional-changelog` package.

Happy to change if that is what is preferred, but the `404` was definitely not useful 🤓

Comment (hbetts) [reply packages/conventional-changelog-cli/README.md]: > the original repo link ajoslin/conventional-changelog

Oh, no, that's actually a good point. Let's stick with the original intent.

We can change the URL to point to the CLI at a later time.

**Comment by eddiejaoude** [comment]
screenshot 2018-02-17 17 38 11


```bash
  conventionalChangelog
    ✓ should overwrite if `-s` presents when appending (3059ms)
    1) should not warn if preset is found
    2) should work with mixed case
    ✓ should warn if preset is not found
    3) should not warn if preset is found
    4) should work with mixed case
    ✓ should still work if preset is not found (513ms)
  2 passing (2s)
  4 failing
  1) conventionalChangelog
       should not warn if preset is found:
     Error: done() called multiple times
      at DestroyableTransform._flush (test/test.js:1:9340)
      at DestroyableTransform.prefinish (node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:138:10)
      at prefinish (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:609:14)
      at finishMaybe (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:617:5)
      at endWritable (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:628:3)
      at DestroyableTransform.Writable.end (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:584:41)
      at Readable.onend (_stream_readable.js:498:10)
      at endReadableNT (_stream_readable.js:923:12)
  2) conventionalChangelog
       should work with mixed case:
     Error: done() called multiple times
      at DestroyableTransform._flush (test/test.js:1:10030)
      at DestroyableTransform.prefinish (node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:138:10)
      at prefinish (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:609:14)
      at finishMaybe (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:617:5)
      at endWritable (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:628:3)
      at DestroyableTransform.Writable.end (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:584:41)
      at Readable.onend (_stream_readable.js:498:10)
      at endReadableNT (_stream_readable.js:923:12)
  3) conventionalChangelog
       should not warn if preset is found:
     Error: done() called multiple times
      at DestroyableTransform._flush (test/test.js:1:9340)
      at DestroyableTransform.prefinish (node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:138:10)
      at prefinish (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:609:14)
      at finishMaybe (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:617:5)
      at endWritable (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:628:3)
      at DestroyableTransform.Writable.end (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:584:41)
      at Readable.onend (_stream_readable.js:498:10)
      at endReadableNT (_stream_readable.js:923:12)
  4) conventionalChangelog
       should work with mixed case:
     Error: done() called multiple times
      at DestroyableTransform._flush (test/test.js:1:10030)
      at DestroyableTransform.prefinish (node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:138:10)
      at prefinish (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:609:14)
      at finishMaybe (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:617:5)
      at endWritable (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:628:3)
      at DestroyableTransform.Writable.end (node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:584:41)
      at Readable.onend (_stream_readable.js:498:10)
      at endReadableNT (_stream_readable.js:923:12)
```

**Comment by hbetts** [comment]
Still looking into the broken build.

The issue is that our Node 4 build uses `npm` version `2`, which symlinks all dependencies directly in the `node_modules/` folder under `conventional-changelog` (Instead of relying on hoisted packages supported by newer versions of `npm`). When the tests execute, they attempt to `require` the `angular` preset, which returns `undefined` because Node 4 can't resolve symlink'd paths.

Therefore the solution, I believe, is to update our build to use `npm` version `3` in our Node 4 build.

Not great, because `npm` version `2` is the default for Node 4 installations, but unfortunately our current mono-repository setup doesn't work in such a way to make testing `npm` version `2` possible.

**Comment by hbetts** [comment]
@eddiejaoude the build is working once again. :+1: Can you please re-base your changes on the latest commit in the `master` branch?

We can then get this change merged into the project. :smile:

**Comment by eddiejaoude** [comment]
Done 👍

**Comment by hbetts** [comment]
@eddiejaoude your documentation change was published in version [`1.3.14`](https://github.com/conventional-changelog/conventional-changelog/releases/tag/conventional-changelog-cli%401.3.14) of `conventional-changelog-cli` so that it would appear on the `npm` registry website - https://www.npmjs.com/package/conventional-changelog-cli

You also inspired me to finally fix the URLs in all the `package.json` files so that they properly point to our mono-repository.

**Pull request merged**